### DISCUSSION TITLE
Azure DevOps support

### DISCUSSION THREAD

**Discussion by alexeik** [gitextensions/gitextensions#8671] [Ideas] [answered]
Will it be ?
1.Pull Request to Azure DevOps

**Comment by RussKie** [comment]
Please elaborate.

**Comment by pmiossec** [comment]
>Will it be ?

Maybe. The day one of the GitExtensions user that need it (like you) will spend the time to add the support of this feature.

Can I remind you that this is an opensource project that only live on contribution. So if you don't do it yourself, you will have to wait (maybe forever) until someone contribute it... 

That's in my todo list (but doing it just for the fun because I don't really need it) but very low behind some other features and somewhat behind:

* Spending some time with family
* Fighting again the couch appeal
* Having some rest/sleep
* My job

So I will be pleased if I see this fixed by someone else...

>Pull Request to Azure DevOps
>Please elaborate.

I think @alexeik wants something like the same support of PR that we already have for GitHub but for Azure DevOps PRs...

**Comment by alexeik** [comment]
Yes, something like GitHub PR , FOrk, functions but for Azure DevOps i mean.

but as i see git has "git request-pull" command
but how to invoke this command from GE?

**Comment by pmiossec** [comment]
>but as i see git has "git request-pull" command
>but how to invoke this command from GE?

As it will output in the console, I think the best way is to use the command line (or perhaps GitExtensions scripts)

But I'm pretty sure you misunderstood the goal of this command. It will not create a pull request (git doesn't know the notion of PR like GitHub or Azure Devops) but will output a summary to send by email (or something else) to a project to ask for pulling your developments...

**Comment by alexeik** [comment]
@pmiossec  maybe u r right.
but maybe Azure can check this request  as own PR? :)

**Comment by pmiossec** [comment, marked as answer]
>but maybe Azure can check this request as own PR? :)

I will let you dream that it is possible....

**Comment by alexeik** [comment]
will check on monday is it dream or not :))
